docs/api-site-config.md

```diff
@@ -151,6 +151,10 @@ h1 {
 
 `separateCss` - Folders inside which any `css` files will not be processed and concatenated to Docusaurus' styles. This is to support static `html` pages that may be separate from Docusaurus with completely separate styles.
 
+`scrollToTopOptions` - Set this to `true` if you want to enable the scroll to top button at the bottom of your site.
```

Comment:
This is incorrect... The fields are swapped

Reply:
Yeah your right... I will fix it !